src/Avalonia.Input/TappedEventArgs.cs

```diff
@@ -6,7 +6,12 @@ namespace Avalonia.Input
     public class TappedEventArgs : RoutedEventArgs
     {
         private readonly PointerEventArgs lastPointerEventArgs;
-
+        public IPointer Pointer => lastPointerEventArgs.Pointer;
```

Comment:
nit: Can you move properties above the constructor? Between ctor and methods.

Reply:
I hope I did everything right :)